### ISSUE TITLE
[suggestion] cycle through placement/breaking restriction modes with scrolling instead of assigning hotkeys for each of the modes

### ISSUE THREAD

**Issue by anaizy31** [maruohon/tweakeroo#123]
^ I really like how we can cycle through the slot number when holding the hotkeys to toggle randomize hotbar etc.
with all the awesome masady mods installed we often need to memorized a ton of hotkeys, this should simply things a little bit.

**Comment by maruohon** [comment]
Good catch, not sure why I didn't think of this before... I'll definitely add this to the placement and breaking restrictions.
